"//caffe2/test/inductor:custom_ops" is a FB-specific "library" which we pull in by setting IS_SANDCASTLE causing
> OSError: /caffe2/test/inductor:custom_ops: cannot open shared object file: No such file or directory
in inductor/test_aot_inductor_custom_ops.py

Author: Alexander Grund (TU Dresden)
diff --git a/test/inductor/test_aot_inductor_custom_ops.py b/test/inductor/test_aot_inductor_custom_ops.py
index ce2ef3739d3..7b9dc4792fd 100644
--- a/test/inductor/test_aot_inductor_custom_ops.py
+++ b/test/inductor/test_aot_inductor_custom_ops.py
@@ -380,7 +380,7 @@ common_utils.instantiate_parametrized_tests(AOTInductorTestsTemplate)
 
 class AOTICustomOpTestCase(TestCase):
     def setUp(self):
-        if IS_SANDCASTLE or IS_FBCODE:
+        if False:
             torch.ops.load_library("//caffe2/test/inductor:custom_ops")
         elif IS_MACOS:
             raise unittest.SkipTest("non-portable load_library call used in test")
